src/sentry/static/sentry/app/components/organizations/homeSidebar.jsx

```diff
@@ -34,17 +36,18 @@ const OrgSettingsMenu = ({access, org, features}) => {
     <div>
       <h6 className="nav-header with-divider">{t('Manage')}</h6>
       <ul className="nav nav-stacked">
-        {access.has('org:read') && (
-          <ListLink to={`${pathPrefix}/members/`}>
-            {t('Members')}&nbsp;
-            {access.has('org:write') &&
-              org.pendingAccessRequests > 0 && (
-                <span className="badge" style={{marginLeft: 5}}>
-                  {org.pendingAccessRequests}
-                </span>
-              )}
-          </ListLink>
-        )}
+        {access.has('org:read') &&
```

Comment:
In our old django templates, it checked for `org:read`, in our `members/` endpoint it requires `member:read`, not sure what is the ideal behavior here.

Reply:
My 2c - makes sense to match the API. Why restrict access to something on the front end to something that the user gets access to in the API?